### PR TITLE
PP-10710 add new table

### PR DESCRIPTION
**Pull request by SandorArpa**
## WHAT YOU DID
- add a new table `idempotency`
- add columns and unique restrictions (gateway_account_id, key)

### CODE REVIEW

**Review by stephencdaly — DISMISSED** [review]
LGTM!

File: src/main/resources/migrations.xml [hunk 1]
```diff
@@ -1930,4 +1930,37 @@
...
+            </column>
+            <column name="created_date" type="timestamp without timezone"
+                    defaultValueComputed="(now() at time zone 'utc')">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+    </changeSet>
+
+    <changeSet id="create gateway_account_id_and_key_idx index on idempotency table" author="">
```
Comment (kbottla) [inline src/main/resources/migrations.xml]: should this id be 'create unique constraint ...'?

Comment (SandorArpa) [reply src/main/resources/migrations.xml]: 👍

File: src/main/resources/migrations.xml [hunk 2]
```diff
@@ -1930,4 +1930,37 @@
...
+            </column>
+            <column name="created_date" type="timestamp without timezone"
+                    defaultValueComputed="(now() at time zone 'utc')">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+    </changeSet>
+
+    <changeSet id="create unique constraint gateway_account_id_and_key_idx index on idempotency table" author="">
```
Comment (kbottla) [inline src/main/resources/migrations.xml]:
```suggestion
    <changeSet id="create unique constraint gateway_account_id_and_key_unique on idempotency table" author="">
```

Comment (SandorArpa) [reply src/main/resources/migrations.xml]: 👍

**Review by kbottla — APPROVED** [review]

**Pull request merged**